pkg/server/bundle/client/updater_test.go

```diff
@@ -41,17 +42,17 @@ func TestBundleUpdater(t *testing.T) {
 	}{
 		{
 			name:        "local bundle not found",
-			trustDomain: "domain.test",
+			trustDomain: trustDomain,
 			err:         "local bundle not found",
 		},
 		{
 			name:        "accepts trust domain id as the trust domain",
```

Comment:
I think we can probably just get rid of this test case? Using the `TrustDomain` type removes the need to check for this i think.